gradle/libs.versions.toml

```diff
@@ -5,14 +5,14 @@
 
 [versions]
 # Gradle plugin versions
-androidGradlePlugin = "8.5.1"
+androidGradlePlugin = "8.6.0"
 protobufPlugin = "0.9.4"
 ktlint = "11.5.0"
 
 # Android SDK versions
-compileSdk = "34"
-minSdk = "34"
-targetSdk = "34"
+compileSdk = "35"
+minSdk = "35"
```

Comment:
Could we leave this at `34` for now please?

Reply:
Done